trio/_core/_multierror.py

```diff
@@ -114,6 +114,9 @@ def push_tb_down(tb, exc, preserved):
     preserved = set()
     new_root_exc = filter_tree(root_exc, preserved)
     push_tb_down(None, root_exc, preserved)
+    # Delete the local functions avoid a reference cycle (see
+    # test_simple_cancel_scope_usage_doesnt_create_cyclic_garbage)
```

Comment:
How important is it to reference the test?  Seems like this comment could get stale easily.

Reply:
Good question! It's possibly still worth the risk, as stale comments can be fixed, but losed context is harder to fix!